x/staking/simulation/operations.go

```diff
@@ -82,29 +82,41 @@ func WeightedOperations(
 		},
 	)
 
-	appParams.GetOrGenerate(cdc, OpWeightMsgCancelUnbondingDelegation, &weightMsgCancelUnbondingDelegation, nil,
-		func(_ *rand.Rand) {
-			weightMsgCancelUnbondingDelegation = simappparams.DefaultWeightMsgDelegate
-		},
-	)
-
-	appParams.GetOrGenerate(cdc, OpWeightMsgTokenizeShares, &weightMsgTokenizeShares, nil,
-		func(_ *rand.Rand) {
-			weightMsgTokenizeShares = DefaultWeightMsgTokenizeShares
-		},
-	)
-
-	appParams.GetOrGenerate(cdc, OpWeightMsgRedeemTokensforShares, &weightMsgRedeemTokensforShares, nil,
-		func(_ *rand.Rand) {
-			weightMsgRedeemTokensforShares = DefaultWeightMsgRedeemTokensforShares
-		},
-	)
-
-	appParams.GetOrGenerate(cdc, OpWeightMsgTransferTokenizeShareRecord, &weightMsgTransferTokenizeShareRecord, nil,
-		func(_ *rand.Rand) {
-			weightMsgTransferTokenizeShareRecord = DefaultWeightMsgTransferTokenizeShareRecord
-		},
-	)
+	// appParams.GetOrGenerate(cdc, OpWeightMsgCancelUnbondingDelegation, &weightMsgCancelUnbondingDelegation, nil,
```

Comment:
Should we either get the simulation tests working or remove the commented portions?

Reply:
get em working, but in a separate PR